### PR TITLE
website(heading): add do/dont about case

### PR DESCRIPTION
**Pull request by shleewhite**
- [x] update examples to all use sentence case
- [x] add do/dont about using sentence case vs title case

Screen Shot 2022-01-26 at 11 38 04 AM

**Contributing to Twilio**

> All third-party contributors acknowledge that any contributions they provide will be made under the same open-source license that the open-source project is provided under.

- [x] I acknowledge that all my contributions will be made under the project's license.

### CODE REVIEW

**Review by TheSisb — APPROVED** [review]

File: packages/paste-website/src/pages/components/heading/index.mdx [hunk 1]
```diff
@@ -185,6 +186,15 @@ It is important to not skip one or more heading levels as it is common for scree
 
 Headings should be used for page headings or content subsections. Headings are broken into six levels, where h1 is the highest level and h6 is the lowest. Avoid skipping heading levels. Use the next lowest heading level to maintain a coherent document outline.
 
+<DoDont>
+  <Do title="Do" body="Use sentence case in headings.">
```
Comment (TheSisb) [inline packages/paste-website/src/pages/components/heading/index.mdx:190]: Have you tried the `center` prop on these? Might look neater

File: packages/paste-website/src/pages/components/heading/index.mdx [hunk 2]
```diff
@@ -82,7 +82,8 @@ Headings should allow users to easily distinguish content sections on the page.
 - Headings should be used with intention that maintains a coherent document outline
 - Don’t use heading tags to resize text. Instead use the `<Text>` primitive
 - Don’t skip headline levels. Always start with `<h1>`, `<h2>`, and so on
-- There can only be one `<h1>` tag per page.
+- There can only be one `<h1>` tag per page
+- Always use sentence case when writing headings, for more information read the [product style guide](/foundations/content/product-style-guide/#sentence-case)
```
Comment (gloriliale) [inline packages/paste-website/src/pages/components/heading/index.mdx:86]: I don't see this bullet point in the preview, though I do see the other changes. Is this just me?

**Review by gloriliale — APPROVED** [review]
Love a good sentence case.

File: packages/paste-website/src/pages/components/heading/index.mdx (hunk 2 above)
Comment (shleewhite) [reply packages/paste-website/src/pages/components/heading/index.mdx:86]: Ooooh - the preview probably hasn't been updated yet

File: packages/paste-website/src/pages/components/heading/index.mdx (hunk 1 above)
Comment (shleewhite) [reply packages/paste-website/src/pages/components/heading/index.mdx]: that looks way better, updated!

**Pull request merged**